Sources/Perception/Perceptible.swift

```diff
@@ -21,3 +21,5 @@
 @available(watchOS, deprecated: 10, renamed: "Observable")
 @available(tvOS, deprecated: 17, renamed: "Observable")
 public protocol Perceptible {}
+
+public protocol Perceptible2: Perceptible {}
```

Comment:
```suggestion

public protocol Perceptible2: Perceptible {}
```

We should remove this right?